app/facades/movie_facade.rb

```diff
@@ -0,0 +1,63 @@
+class MovieFacade
+
+    def self.top_movies_endpoint(page_num)
+        sort_by = 'vote_average.desc'
+        conn.get("3/discover/movie?api_key=#{movies_api_key}&#{language('en-US')}&sort_by=#{sort_by}&#{exclude_adult}&page=#{page_num}&vote_count.gte=300")
+    end
+
+    def self.keyword_search_endpoint(page_num, keywords)
+        conn.get("3/search/movie?api_key=#{movies_api_key}&#{language('en-US')}&#{exclude_adult}&page=#{page_num}&query=#{keywords}")
+    end
+
+    def self.keyword_search(keywords)
+        response1 = keyword_search_endpoint(1, keywords)
+        response2 = keyword_search_endpoint(2, keywords)
+
+        json1 = parse(response1)
+        json2 = parse(response2)
+        movies1 = json1[:results].map do |movie_data|
+            MovieCreator.new(movie_data)
+        end
+        movies2 = json2[:results].map do |movie_data|
+            MovieCreator.new(movie_data)
+        end
+        (movies1.concat(movies2)).flatten
+    end
+
+    def self.top_movies
+        response1 = top_movies_endpoint(1)
+        response2 = top_movies_endpoint(2)
+
+        json1 = parse(response1)
+        json2 = parse(response2)
+        movies1 = json1[:results].map do |movie_data|
+            MovieCreator.new(movie_data)
+        end
+        movies2 = json2[:results].map do |movie_data|
+            MovieCreator.new(movie_data)
+        end
+        (movies1.concat(movies2)).flatten
```

Comment:
@dhalverson If time, we can refactor this to loop through X num of times (in our case, 2). I'll create a ticket now.